lib/galaxy/tool_util/ontologies/ontology_data.py

```diff
@@ -0,0 +1,100 @@
+from typing import (
+    cast,
+    Dict,
+    List,
+    NamedTuple,
+    Optional,
+    Tuple,
+)
+
+from pkg_resources import resource_string
```

Comment:
Can you use `importlib_resources` for this ?
https://importlib-resources.readthedocs.io/en/latest/migration.html#pkg-resources-resource-string